### PR TITLE
Make firecracker setup configurable

### PR DESCRIPTION
**Pull request by ganeshmaharaj**

### CODE REVIEW

**Comment by krsna1729** [comment]
Default case works as expected. Will the env variable to turn on FC be added in a follow on?
/cc @mcastelino 
```
Events:
  Type     Reason                  Age    From               Message
  ----     ------                  ----   ----               -------
  Normal   Scheduled               3m29s  default-scheduler  Successfully assigned default/php-apache-kata-fc-6c6f484c4b-dgxqs to clr-03
  Warning  FailedCreatePodSandBox  3m8s   kubelet, clr-03    Failed create pod sandbox: rpc error: code = Unknown desc = container create failed: rpc error: code = Unknown desc = rootfs (/run/kata-containers/shared/containers/1cdfc6d9889e28388ff6ad3e256262cd05984371e9676982ffb4dbfca05b6a4b/rootfs) does not exist
```

```
Every 2.0s: kubectl get pods --all-namespaces -owide                                                                                                            clr-01: Thu Mar 21 23:32:30 2019

NAMESPACE     NAME                                    READY   STATUS              RESTARTS   AGE     IP                NODE     NOMINATED NODE   READINESS GATES
default       php-apache-kata-fc-6c6f484c4b-dgxqs     0/1     ContainerCreating   0          81s     <none>            clr-03   <none>           <none>
default       php-apache-kata-qemu-7bc9ff655b-89jr9   1/1     Running             0          81s     10.244.2.4        clr-03   <none>           <none>
default       php-apache-runc-7cc9dcb54f-km7xw        1/1     Running             0          81s     10.244.1.6        clr-02   <none>           <none>
kube-system   canal-b8trq                             2/3     Running             1          3m11s   192.168.121.165   clr-03   <none>           <none>
kube-system   canal-csfsw                             2/3     Running             2          3m11s   192.168.121.140   clr-02   <none>           <none>
kube-system   canal-t4h4z                             2/3     Running             2          3m11s   192.168.121.105   clr-01   <none>           <none>
kube-system   coredns-86c58d9df4-64kgq                1/1     Running             0          4m11s   10.244.1.3        clr-02   <none>           <none>
kube-system   coredns-86c58d9df4-78652                1/1     Running             0          4m11s   10.244.1.4        clr-02   <none>           <none>
kube-system   etcd-clr-01                             1/1     Running             0          3m10s   192.168.121.105   clr-01   <none>           <none>
kube-system   kata-deploy-nmkrg                       1/1     Running             0          2m51s   10.244.2.2        clr-03   <none>           <none>
kube-system   kata-deploy-nxkkk                       1/1     Running             0          2m52s   10.244.1.5        clr-02   <none>           <none>
kube-system   kube-apiserver-clr-01                   1/1     Running             0          3m29s   192.168.121.105   clr-01   <none>           <none>
kube-system   kube-controller-manager-clr-01          1/1     Running             0          3m19s   192.168.121.105   clr-01   <none>           <none>
kube-system   kube-proxy-6p27x                        1/1     Running             0          3m42s   192.168.121.140   clr-02   <none>           <none>
kube-system   kube-proxy-d8vkz                        1/1     Running             0          3m21s   192.168.121.165   clr-03   <none>           <none>
kube-system   kube-proxy-mpljm                        1/1     Running             0          4m11s   192.168.121.105   clr-01   <none>           <none>
kube-system   kube-scheduler-clr-01                   1/1     Running             0          3m24s   192.168.121.105   clr-01   <none>           <none>
kube-system   metrics-server-5465bdb788-bhrw7         1/1     Running             0          3m9s    10.244.1.2        clr-02   <none>           <none>
```

**Comment by ganeshmaharaj** [comment]
Lol. I missed that part didn't i? This is why I shouldn't rush it. Will fix
it tonight.

File: clr-k8s-examples/Vagrantfile [hunk 1]
```diff
@@ -10,17 +10,16 @@ $cpus = (ENV['CPUS'] || 2).to_i
...
-base_ip = IPAddr.new("192.52.100.10")
-hosts = {}
-proxy_ip_list = ""
-#DISK_UUID = Time.now.utc.to_i
-driveletters = ('a'..'z').to_a
+$base_ip = IPAddr.new("192.52.100.10")
+$hosts = {}
+$proxy_ip_list = ""
+$driveletters = ('a'..'z').to_a
+$setup_fc = false
```
Comment (krsna1729) [inline clr-k8s-examples/Vagrantfile]: ENV here :)

**Comment by ganeshmaharaj** [comment]
@krsna1729 should be fixed now. Should we land it?

**Comment by krsna1729** [comment]
@ganeshmaharaj forgot to ask for doc update. Can you please send follow up PR

**Pull request merged**